apps/web/ee/lib/stripe/customer.ts

```diff
@@ -46,6 +46,21 @@ export async function getStripeCustomerId(user: UserType): Promise<string | null
     });
```

Comment:
There's a missing edge-case here. If we don't have the stripeCustomerId in our DB but it exists in Stripe, we need to save it in our DB as well.

Reply:
Just moved prisma user update to run for those 2 scenarios(Get from stripe and not on our db and newly created stripe.Customer).